src/functions/mod.ts

```diff
@@ -1,9 +1,16 @@
-import { ManifestFunctionSchema } from "../manifest/manifest_schema.ts";
+import {
+  ManifestFunctionSchema,
+  ManifestFunctionType,
+} from "../manifest/manifest_schema.ts";
 import {
   ParameterSetDefinition,
   PossibleParameterKeys,
 } from "../parameters/types.ts";
-import { FunctionDefinitionArgs } from "./types.ts";
+import {
+  FunctionDefinitionArgs,
```

Comment:
nit: Should we rename to `BaseFunctionDefinitionArgs`?

Reply:
I'm not a huge fan of the `base` prefix, if we don't need it I would try not to use it

Reply:
Yeah the `base` prefix is an established pattern we use pretty commonly in this repo and others to indicate a parent class. In this case my thinking was that because these are the args for `BaseFunctionDefinition`, not necessarily the args for `SlackFunctionDefinition` or `ConnectorFunctionDefinition`, and it seems nice to indicate as such in the name

Reply:
Makes sense, I've renamed the type 🟢

Reply:
Renaming `FunctionDefinitionArgs` to `BaseFunctionDefinitionArgs` introduced many changes to in interactivity folder which does not seem necessary
I would potentially want to rename the following 

- `BaseFunctionDefinition` -> `FunctionDefinition`
- `BaseFunctionDefinitionArgs` -> `FunctionDefinitionArgs`

I feel like this is more descriptive and introduces less change in the PR, I will implement it now to keep the PR clean let me know what you think